src/resources/postcss/views/day/_event.pcss

```diff
@@ -0,0 +1,86 @@
+.tribe-events {
+
+	/* -----------------------------------------------------------------------------
+	 * Event
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-events-calendar-day__event {
+		display: flex;
+		flex-direction: column;
+		margin-bottom: var(--spacer-5);
+		margin-top: var(--spacer-8);
+
+		@media (--viewport-medium) {
+			flex-direction: row-reverse;
+			justify-content: flex-end;
+			margin-bottom: var(--spacer-7);
+			margin-top: var(--spacer-10);
+		}
+
+		&:last-child {
+			margin-bottom: var(--spacer-8);
+
+			@media (--viewport-medium) {
+				margin-bottom: var(--spacer-10);
+			}
+		}
+	}
+
+	.tribe-events-calendar-day__event-featured-image-wrapper {
+		flex: none;
+		margin-bottom: var(--spacer-5);
+
+		@media (--viewport-medium) {
+			margin-bottom: 0;
+			width: calc(3 * var(--grid-width-1-of-9));
+		}
+	}
+
+	.tribe-events-calendar-day__event-featured-image {
+		padding-bottom: 56.25%;
+	}
+
+	.tribe-events-calendar-day__event-details {
+		flex: none;
+
+		@media (--viewport-medium) {
+			margin-left: var(--grid-width-1-of-9);
+			width: calc(5 * var(--grid-width-1-of-9));
+		}
+	}
+
+	.tribe-events-calendar-day__event-datetime-wrapper {
```

Comment:
Let's add https://github.com/moderntribe/the-events-calendar/pull/2583/files#diff-ff3de376ecb4382618c6419ad7a5b4f4R101 before this, otherwise we run into some funny spacing if we don't have a description.